### ISSUE TITLE
systemd config

### ISSUE THREAD

**Issue by subogero** [subogero/omxd#51]
Add systemd unit file and install.

**Comment by subogero** [comment]
Released in version 1.7

**Comment by OMID-313** [comment]
Thanks @subogero for adding systemd compatibility.
Can we get the new version of omxd via `apt-get` !?

**Comment by subogero** [comment]
Yes, version 1.7 is in my deb repo.

**Comment by OMID-313** [comment]
I installed the new version of omxd. But it seems that it has a problem. Because putting omxd in rc.local to start playing videos after boot, results in simultaneous playing of video number 1 and 2.
Can you reproduce the bug @subogero ?

**Comment by OMID-313** [comment]
I tried running the code outside of rc.local . The same problem happens. Videos 1 and 2 start to play simultaneously, and after a few seconds, the screen freezes.
This is my code:

```
omxd X
omxd j
omxd O --no-osd
omxd A /mfolder
sleep 1.5
omxd -
sleep 0.1
omxd -
sleep 0.1
omxd -
sleep 0.1
omxd -
sleep 0.1
omxd -
```

**Comment by OMID-313** [comment]
Dear @subogero ,
The problem outside rc.local solved by changing `sleep 1.5` to `sleep 2` in the above code.
But the `rc.local` problem still is happening.
I think the rc.local is acting differently in jessie than wheezy. Because the video starts to play (though simultaneously), and then gets stopped by the login command.

**Comment by OMID-313** [comment]
The rc.local problem solved by changing `sleep 2` to `sleep 3` :D
